### PR TITLE
[FEATURE] pad - string padding

### PR DESCRIPTION
**Pull request by commenthol**
Adds string padding function.
- `pad` pads on both sides

## What does your PR belong to?
- [ ] Website
- [x] Snippets
- [ ] General / Things regarding the repository (like CI Integration)
- [ ] Tests

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (non-breaking improvement of a snippet)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have checked that the changes are working properly
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

**Comment by Chalarangelo** [comment]
These methods are natively implemented, so I don't think we should add them.

**Comment by commenthol** [comment]
Did not know so far. Thanks for telling me. 
Albeit `pad` is not available. Do you want me to leave just the pad  function?

**Comment by Chalarangelo** [comment]
I'm away for the weekend, so I cannot review it right now, but if we keep anything it will just be the `pad` snippet. If any other members of the core team can review it and provided that this is not natively implemented, I suppose we should keep it.

File: snippets/pad.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+### pad
+
+Pads `string` on the left and right side if it's shorter than `length`.
+String is *NOT* truncated on both sides if it exceeds length.
+
+```js
+const pad = (string, length = 8, char = ' ') =>
+  string.padStart(string.length + (length - string.length) / 2, char).padEnd(length, char);
```
Comment (kriadmin) [inline snippets/pad.md]: `string.length/2` should be `Math.floor(string.length/2)` just to be safe against people looking for fun.

File: snippets/pad.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+module.exports = pad;
```
Comment (kriadmin) [inline snippets/pad.md]: Remove this. We don't need it.

File: snippets/pad.md (hunk 1 above)
Comment (commenthol) [reply snippets/pad.md]: Could you give me an example what "fun" I might look for? Just being curious...

**Pull request merged**